recipes/pymmcore/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "pymmcore" %}
+{% set version = "10.1.1.70.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pymmcore-{{ version }}.tar.gz
+  sha256: 7cb9ed0aac35ce4dbaaff8b967d41205bc98b15c621b931d4a2a75474c6d37b0
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - boost-cpp
+    - numpy
+    - pip
+    - python
+    - swig
+  run:
+    - boost-cpp
+    - python
+    - {{ pin_compatible('numpy') }}
+
+test:
+  imports:
+    - pymmcore
+  commands:
+    - pip list
+    - pip check
+    - python -c "import pymmcore;mmc = pymmcore.CMMCore();print(mmc)"
+  requires:
+    - pip
+
+about:
+  home: https://github.com/micro-manager/pymmcore
+  summary: Python bindings for MMCore, Micro-Manager's device control layer
+  license: LGPL-2.1
+  license_file: LICENSE.txt
```

Comment:
I see the repo has the mmcore submodule. I think it's only used to generate the bindings so there's no code being vendored in the final release, but I wonder if we still need to include the upstream license here for completeness?

Reply:
@marktsuchida @tlambert03

Reply:
The C++ code in the `MMDevice` and `MMCore` directories of `mmCoreAndDevices` gets built into the Python extension module. The licenses are BSD-3 and LGPL-2.1, respectively. (Each of the 2 directories contains a `license.txt` file.)

We take this approach because the build system of `mmCoreAndDevices` does not officially produce a C++ library for MMCore (only MMCoreJ, the Java bindings, is released).

Reply:
In that case we need to add the licenses, yes. Thanks for the reply!